README.md

```diff
@@ -1,8 +1,32 @@
 # SymPy Paper
 
-[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+This is the source repository for the paper:
+
+> SymPy Authors, "SymPy: Symbolic Computing in Python", 2016.
+
+This repository contains or links to all of the information needed to reproduce
+the results in the paper.
+
+The latest rendered version of the PDF can be viewed via the
+[ShareLaTeX](http://sharelatex.com) CI system:
 
 [![PDF Status](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/badge.svg)](https://www.sharelatex.com/github/repos/asmeurer/sympy-paper/builds/latest/output.pdf)
 
+# Contributing
+
+Join our chat room to discuss contributions:
+
+[![Join the chat at https://gitter.im/asmeurer/sympy-paper](https://badges.gitter.im/asmeurer/sympy-paper.svg)](https://gitter.im/asmeurer/sympy-paper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+# License
+
+The contents of this repository is licensed under the [Creative Commons
+Attribution 4.0 International
+License](http://creativecommons.org/licenses/by/4.0).
+
+[![CCBY](https://i.creativecommons.org/l/by/4.0/80x15.png)](http://creativecommons.org/licenses/by/4.0)
```

Comment:
Sorry for my bad grammar. I meant to write: is CC-BY an open source license? (I fixed it above.) So your question might be: "what is an open source license"? The answer is that an open source license must have a few attributes in order to be considered an open source license, they are listed here:

https://opensource.org/osd

This list was actually adapted from the Debian Free Software Guidelines (DFSG): https://www.debian.org/social_contract#guidelines, they are almost the same thing. So if a given license is not DFSG free, as determined by consensus on the [debian-legal](https://www.debian.org/legal/) mailinglist (or, for practical purposes almost the same thing, nor approved by opensource.org), then it is not an open source license.

CC-BY, that you like, was **not** an open source license, as of version 1.0 (see my link above for details). So I am strongly against using that. Version 4.0 (the version you are actually proposing) **might** be an open source license, but I wasn't able to find it clearly approved by either DFSG or opensource.org. 

@moorepants and @scopatz asked, why it matters to open source this repository? (I can't believe I have to lobby to open source things here....) But here we go: Because we, or somebody else might want to include this article perhaps with the rest of the documentation in the `sympy-doc` package, perhaps in Debian. Unless the sources of the pdf are released under an open source license, then Debian will not distribute it.

It is the same problem as with the GNU Free Documentation License (GFDL), which is not [an open source license](https://wiki.debian.org/DFSGLicenses#GNU_Free_Documentation_License_.28GFDL.29), so it is not distributed by Debian (yes, there are some finer points regarding "invariant sections", but let's not go into it here).

@scopatz raised an implied question why we would use a software license for this pdf? Because the sources are in latex, which is essentially just a software program. For example I would like to be *allowed to* use parts of it in sympy documentation later (maybe we will, maybe we won't, but I want to have the option) --- for example the limit algorithm that I described in great detail here. If we used a license CC-BY 1.0 for this repository, we copied some stuff into sympy documentation, then Debian would have to strip it, otherwise they wouldn't be able to distribute it. What a terrible and messy thing to do by us. I am obviously strongly against it.

I know you are advocating CC-BY 4.0, but show me that it's an indisputably open source license, then I might consider it.

@scopatz mentioned that "it's pretty much unheard of to use a software licence." In fact, that is what Debian recommends since 2006, see here: https://www.debian.org/vote/2006/vote_001 where they made the exact same points I just made above, see the section "(3) Why does documentation need to be Free Software?":

>  There are a number of obvious differences between programs and documentation that often inspire people to ask "why not simply have different standards for the two?" For example, books are often written by individuals, while programs are written by teams, so proper credit for a book might be more important than proper credit for a program. 

... (read the answer at the link I provided)

Finally, Debian recommends (also on that link):

> As a consequence, other licenses should be investigated; generally it is probably simplest to choose either the GNU General Public License (for a copyleft license) or the BSD or MIT licenses (for a non-copyleft license). 



Conclusion: the general accepted practice in open source (Debian) is to use open source software licenses for both codes, as well as documentation and books.

I have indeed seen many book authors to use CC-BY 4.0, so perhaps it is an open source license. However, to make things simple, I propose we dual license this repository under the SymPy BSD license (or MIT) and CC-BY 4.0, if you think that's a better license (I am not convinced that it's a better license).